frontend/integration-tests/tests/olm/prometheus.scenario.ts

```diff
@@ -117,9 +117,9 @@ describe('Interacting with the Prometheus OCS', () => {
 
   it('displays the raw YAML for the `Prometheus`', async() => {
     await element(by.linkText('YAML')).click();
-    await browser.wait(until.presenceOf($('.yaml-editor--buttons')));
-    await $('.yaml-editor--buttons').element(by.buttonText('Save')).click();
-    await browser.wait(until.visibilityOf(crudView.successMessage), 1000);
+    await browser.wait(until.presenceOf($('.yaml-editor__buttons')));
+    await $('.yaml-editor__buttons').element(by.buttonText('Save')).click();
+    await browser.wait(until.visibilityOf($('.alert-success')), 1000);
```

Comment:
We should rollback the change to this line

Reply:
I can make the change when I am adding drag and drop